src/Common/tests/Common.Tests.csproj

```diff
@@ -139,48 +139,48 @@
+      <Link>System.Private.CoreLib\src\System\IO\PathInternal.cs</Link>
     </Compile>
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsWindows)'=='true'">
-    <Compile Include="$(CommonPath)\CoreLib\System\IO\PathInternal.Windows.cs">
-      <Link>Common\CoreLib\System\IO\PathInternal.Windows.cs</Link>
+    <Compile Include="$(CoreLibDir)System\IO\PathInternal.Windows.cs">
+      <Link>System.Private.CoreLib\src\System\IO\PathInternal.Windows.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
       <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
     </Compile>
     <Compile Include="Tests\System\IO\PathInternal.Windows.Tests.cs" />
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Kernel32\Interop.FormatMessage.cs">
+    <Compile Include="$(CoreLibDir)Interop\Windows\Kernel32\Interop.FormatMessage.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.FormatMessage.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Windows\Interop.Errors.cs">
       <Link>Common\Interop\Windows\Interop.Errors.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\System\IO\Win32Marshal.cs">
-      <Link>Common\CoreLib\System\IO\Win32Marshal.cs</Link>
+    <Compile Include="$(CoreLibDir)System\IO\Win32Marshal.cs">
+      <Link>System.Private.CoreLib\src\System\IO\Win32Marshal.cs</Link>
     </Compile>
     <Compile Include="Tests\System\IO\Win32Marshal.Tests.cs" />
-    <Compile Include="$(CommonPath)\CoreLib\System\PasteArguments.Windows.cs">
-      <Link>Common\CoreLib\System\PasteArguments.Windows.cs</Link>
+    <Compile Include="$(CoreLibDir)System\PasteArguments.Windows.cs">
+      <Link>System.Private.CoreLib\src\System\PasteArguments.Windows.cs</Link>
     </Compile>
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsUnix)'=='true'">
     <Compile Include="Tests\System\IO\PathInternal.Unix.Tests.cs" />
-    <Compile Include="$(CommonPath)\CoreLib\System\IO\PathInternal.Unix.cs">
-      <Link>Common\CoreLib\System\IO\PathInternal.Unix.cs</Link>
+    <Compile Include="$(CoreLibDir)System\IO\PathInternal.Unix.cs">
+      <Link>System.Private.CoreLib\src\System\IO\PathInternal.Unix.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Unix\System.Native\Interop.PathConf.cs">
+    <Compile Include="$(CoreLibDir)Interop\Unix\System.Native\Interop.PathConf.cs">
       <Link>Common\Interop\Unix\Interop.PathConf.cs</Link>
     </Compile>
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Common\Interop\Unix\Interop.Libraries.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\System\PasteArguments.Unix.cs">
-      <Link>Common\CoreLib\System\PasteArguments.Unix.cs</Link>
+    <Compile Include="$(CoreLibDir)System\PasteArguments.Unix.cs">
+      <Link>System.Private.CoreLib\src\System\PasteArguments.Unix.cs</Link>
```

Comment:
@stephentoub @jkotas do we actually want System.Private.CoreLib\src in the Link paths or just System\...?

Reply:
I decided to remove those.